webapp/src/components/systemConsole/surveyDateTime/surveyDateTime.tsx

```diff
@@ -1,86 +1,61 @@
+import {format} from 'date-fns';
+import {formatInTimeZone} from 'date-fns-tz';
 import React, {useCallback, useEffect, useMemo, useState} from 'react';
+import utils from 'utils/utils';
 
 import SurveyDateSelector from 'components/surveyDateSelector/surveyDateSelector';
 import SurveyTimeSelector from 'components/surveyTimeSelector/surveyTimeSelector';
 import type {CustomSettingChildComponentProp} from 'components/systemConsole/index';
 
-import type {DateTimeConfig} from 'types/plugin';
-
 import './style.scss';
-
-const DEFAULT_SURVEY_TIME = '00:00';
+import type {DateTimeConfig} from 'types/plugin';
 
 function SurveyDateTime({id, setSaveNeeded, onChange, config, setInitialSetting}: CustomSettingChildComponentProp) {
-    const [surveyTime, setSurveyTime] = useState<string>(DEFAULT_SURVEY_TIME);
     const [surveyDate, setSurveyDate] = useState<Date>();
 
-    const [surveyDateTime, setSurveyDateTime] = useState<Date>();
+    // sets the date picker and time picker to the values saved in config on load
+    useEffect(() => {
+        const dateTimeConfig = config.PluginSettings.Plugins['com.mattermost.user-survey']?.systemconsolesetting?.SurveyDateTime;
 
-    useEffect(() => {// Extract date components from the UTC Date object
-        if (!surveyDate) {
-            return;
+        if (dateTimeConfig?.timestamp) {
+            setSurveyDate(utils.unixTimestampToDate(dateTimeConfig.timestamp));
         }
 
-        const year = surveyDate.getFullYear();
-        const month = surveyDate.getMonth(); // getUTCMonth() returns month index (0-11)
-        const day = surveyDate.getDate();
-
-        // Parse time components from timeString
-        const [hours, minutes] = surveyTime.split(':').map(Number);
-
-        // Construct a new Date object in UTC with the combined date and time
-        // Convert the UTC Date object to the user's local time zone
-        setSurveyDateTime(new Date(Date.UTC(year, month, day, hours, minutes)));
-    }, [surveyDate, surveyTime]);
+        setInitialSetting(id, {timestamp: dateTimeConfig?.timestamp});
+    }, [config.PluginSettings.Plugins, id, setInitialSetting]);
 
-    // sets the date picker and time picker to the values saved in config on load
-    useEffect(() => {
-        const dateTimeConfig = config.PluginSettings?.Plugins?.['com.mattermost.user-survey']?.systemconsolesetting?.SurveyDateTime;
+    const helpText = useMemo(() => {
+        let line1 = '';
 
-        const initialConfig: DateTimeConfig = {
-            time: DEFAULT_SURVEY_TIME,
-        };
+        if (surveyDate) {
+            const localDateString = format(surveyDate, "MMMM d, yyy 'at' HH:mm O");
+            const utcDateString = formatInTimeZone(surveyDate, 'UTC', "MMMM d, yyy 'at' HH:mm 'UTC'");
 
-        if (dateTimeConfig?.time) {
-            setSurveyTime(dateTimeConfig.time);
-            initialConfig.time = dateTimeConfig.time;
+            line1 = `Scheduled for ${localDateString} (${utcDateString})\n\n`;
```

Comment:
0/5 since this is an issue with the existing code, but I think these lines should technically be two different `<p>` tags rather than one tag with newlines in it

Reply:
Done.